From 10e818de5a0bdf523d83d6179c0d85a6df1a2d2e Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:20:28 -0400
Subject: [alsa] hda: Add QEMU codec vendor ID

Message-id: <140972823573-130-patchset-jkysela@redhat.com>
Patchwork-id: 90830
O-Subject: [RHEL 7.1 PATCH 129/442] ALSA: hda - Add QEMU codec vendor ID
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: dff86f86d43f17db8616d06f4150c8957d33864b)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/hda_codec.c b/sound/pci/hda/hda_codec.c
index 4be322b..1155027 100644
--- a/sound/pci/hda/hda_codec.c
+++ b/sound/pci/hda/hda_codec.c
@@ -67,6 +67,7 @@ static struct hda_vendor_id hda_vendor_ids[] = {
  { 0x17e8, "Chrontel" },
  { 0x1854, "LG" },
  { 0x1aec, "Wolfson Microelectronics" },
+ { 0x1af4, "QEMU" },
  { 0x434d, "C-Media" },
  { 0x8086, "Intel" },
  { 0x8384, "SigmaTel" },
-- 
1.7.1